storage/store.go

```diff
@@ -0,0 +1,28 @@
+package storage
+
+type KVStore interface {
+    // key value
+    Put(key string, value string) error
+    Get(key string) (string, error)
+    Del(key string) error
+    Contains(key string) bool
```

Comment:
What we need is hierarchical. Starting from top-level `tasks` and then a directory/container `task-id`, and then a map like: `content` = task detailed information, `stats` = task runtime information, `cancellation` = cancellation request etc. How does this hierarchy reflected in the interface?